### PR TITLE
Open a terminal with an wp alias to the docker run command

### PR DESCRIPTION
**Pull request by draganescu**
This PR is for #96 and it adds a button that opens a Terminal window on OSX and executes an alias command that gives easy access to the `wp` command.

### CODE REVIEW

File: src/components/status-panel/index.js [hunk 1]
```diff
@@ -30,11 +30,11 @@ class StatusPanel extends Component {
 				</div>
 			),
 			intermediary: (
-				<p>We're building your environment right now, just sit back and we'll let you know when it's all ready.</p>
+				<p>We&quot;re building your environment right now, just sit back and we&quot;ll let you know when it&quot;s all ready.</p>
```
Comment (pento) [inline src/components/status-panel/index.js]:
```suggestion
				<p>We&#8217;re building your environment right now, just sit back and we&8217;ll let you know when it&8217;s all ready.</p>
```

WordPress uses curly apostrophes. 🙂

File: src/components/status-panel/index.js [hunk 2]
```diff
@@ -30,11 +30,11 @@ class StatusPanel extends Component {
...
 			),
 			negative: (
 				<div>
-					<p>Something went wrong, here's what we know. Please send this info to your friendly TestPress support crew.</p>
+					<p>Something went wrong, here&quot;s what we know. Please send this info to your friendly TestPress support crew.</p>
```
Comment (pento) [inline src/components/status-panel/index.js]:
```suggestion
					<p>Something went wrong, here&8217;s what we know. Please send this info to your friendly TestPress support crew.</p>
```

File: src/components/status-panel/index.js [hunk 3]
```diff
@@ -72,6 +73,16 @@ class StatusPanel extends Component {
 
 		const url = 'http://localhost:' + preferences.site.port;
 
+		async function openWPCLITerminal() {
+			await spawn( 'osascript', [ '-e', '"tell application \\"Terminal\\"\n ' +
+				' activate\n ' +
+				' set currentTab to do script (\\"alias wp=\'docker-compose run --rm cli \'\\")\n ' +
+				' delay 2\n ' +
+				' do script (\\"cd ' + preferences.basic[ 'gutenberg-folder' ] + ' \\") in currentTab\n ' +
```
Comment (pento) [inline src/components/status-panel/index.js]:
```suggestion
				' do script (\\"cd ' + preferences.basic[ 'wordpress-folder' ] + ' \\") in currentTab\n ' +
```

This looks like it should be `wordpress-folder`?

Comment (draganescu) [reply src/components/status-panel/index.js]: this is gutenberg folder because that is where we have the cli docker container :)

Comment (pento) [reply src/components/status-panel/index.js]: Oh, I see. That's the Gutenberg CLI docker container, not the TestPress CLI container. The latter is in `~/Library/Application Support/testpress/tools`.

**Pull request merged**